server/src/modules/campaign-monitor/campaign-monitor.helper.ts

```diff
@@ -91,7 +96,7 @@ export class CampaignMonitorHelper {
 				},
 				{
 					Key: 'vakken',
-					Value: vakken || null,
+					Value: vakken?.toString() || null,
```

Comment:
I prefer join since it more clearly states how you convert the array to a string
also fix for optIns
```suggestion
					Value: vakken?.join(',') || null,
```